### DISCUSSION TITLE
有时候想知道一个记录究竟是从哪个上游返回的

### DISCUSSION THREAD

**Discussion by mkanako** [IrineSistiana/mosdns#60] [General]
目前除了开debug log，好像没别的方法了，但就是麻烦

于是就想能不能在返回记录里附加点备注之类的信息，然后看了dns 文报格式，好像没啥合适的字段放东西？

不知道各位有没有这种需求的，讨论讨论

**Comment by IrineSistiana** [comment]
“需求”有是有，但太不常用了。我是 ```grep``` 解决。

edns0 padding 可以随便赛任意大小的数据。

**Reply by mkanako** [reply]
是不常用，但想用的时候就觉得烦
`edns0 padding` get 到关键词😎
